pkgs/by-name/gi/git-autoshare/package.nix

```
@@ -0,0 +1,37 @@
+{
+  fetchFromGitHub,
+  git,
+  lib,
+  python3Packages,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "git-autoshare";
+  version = "1.0.0b6";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "acsone";
+    repo = "git-autoshare";
+    rev = version;
+    hash = "sha256-F8wcAayIR6MH8e0cQSwFJn/AVSLG3tVil80APjcFG/0=";
+  };
+
+  build-system = with python3Packages; [setuptools-scm];
+  dependencies = with python3Packages; [appdirs click pyyaml];
+
+  # Tests require network
+  doCheck = false;
+
+  makeWrapperArgs = [
+    "--set-default GIT_AUTOSHARE_GIT_BIN ${lib.getExe git}"
+  ];
+
+  pythonImportsCheck = [ "git_autoshare" ];
+  meta = with lib; {
+    changelog = "https://github.com/acsone/git-autoshare/releases/tag/${version}";
+    description = "Git clone wrapper that automatically uses --reference to save disk space and download time.";
```

*Not end with a period (or any punctuation for that matter).*
> https://github.com/NixOS/nixpkgs/tree/master/pkgs#meta-attributes

```suggestion
    description = "Git clone wrapper that automatically uses --reference to save disk space and download time";
```